src/components/ChatInput.tsx

```diff
@@ -98,12 +98,12 @@ export function ChatInput({
     (e: React.KeyboardEvent<HTMLTextAreaElement>) => {
       if (!e.shiftKey && e.key === "Enter") {
         e.preventDefault();
-        if (canSendMessage) {
+        if (canSendMessage && input.length !== 0) {
```

Comment:
can you add a small test to ChatInput.test.tsx for this?

Reply:
done!